src/Core/Scheduler/Providers/PointCloudProvider.js

```diff
@@ -0,0 +1,282 @@
+import * as THREE from 'three';
+import Provider from './Provider';
+import Fetcher from './Fetcher';
+import PointsMaterial from '../../../Renderer/PointsMaterial';
+
+function PointCloudProvider() {
+    Provider.call(this);
+}
+
+PointCloudProvider.prototype = Object.create(Provider.prototype);
+PointCloudProvider.prototype.constructor = PointCloudProvider;
+
+// PointCloud
+// from potree
+function createChildAABB(aabb, childIndex) {
```

Comment:
A[xis]A[ligned]B[ounding]B[ox]

It's related to PotreeConverter file structure. And this specific function compute the bounding-box of a node's child based on its index.